package.json

```diff
@@ -83,11 +81,6 @@
       "providers/*"
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependents"
-    }
-  },
```

Comment:
Just caused issues for multiple developers, currently disabled it. We will review this later on a more stable solution for running recommit. Perhaps event with NX